.codeclimate.yml

```diff
@@ -0,0 +1,15 @@
+engines:
+  duplication:
+    enabled: true
+    config:
+      languages:
+        javascript:
+          mass_threshold: 100
```

Comment:
I'm wondering how mass 100 compares to the default we have now. It's not clear to me what value we want, esp. given how mass is defined, but if I'm guessing this will dial down the sensitivity a bit?

From https://docs.codeclimate.com/v1.0/docs/duplication#section-understand-the-engine: "Mass" refers to the size of the duplicated code. Specifically, mass is determined by the size of a code block's s-expression, after it has been parsed into a node of an Abstract Syntax Tree (AST).